ravenframework/CodeInterfaceClasses/RELAP5/RELAPparser.py

```diff
@@ -431,10 +440,18 @@ def modifyOrAdd(self,modifyDict,save=True):
         temp.append('*'+' deckNum: '+str(deckNum)+'\n')
       for j in sorted(modiDictionaryList):
         for var in modiDictionaryList[j]:
+          ff = '{:7e}'
+          cast = float
+          if j in intDT and int(intDT[j]) == int(var['position']):
```

Comment:
since we have two datatypes, should we specify only one of the datatypes in the input files?

Reply:
yeah..since float is the default, I remover the input for them (redundant)